LICENSES_AND_LICENSE_HEADERS.md

```diff
@@ -0,0 +1,18 @@
+## LICENSES AND LICENSE HEADERS
+
+OpenSearch-project uses Apache License v2 for all the projects.  
+Refer to [LICENSE](LICENSE)
```

Comment:
Link `[Apache License v2](LICENSE)` instead of "refer", it's the actual license.

Reply:
Thats true :), sure.